extensions/amp-story/1.0/amp-story.js

```diff
@@ -465,6 +465,11 @@ export class AmpStory extends AMP.BaseElement {
         'story-load-first-page-only'
       );
     }
+    if (isExperimentOn(this.win, 'story-disable-animations-first-page')) {
```

Comment:
Some thoughts on this:
- The experiment needs to be enabled for CSI tracking BEFORE the LCP metric is sent. Is it the case here?
- There are more cases where the animations are disabled (prefers-reduced-motion), I think it'd be interesting to capture this as well

Reply:
For 1: This is added on the `buildCallback` of a story, so it should happen before the LCP metric is sent.
For 2: We can trigger the CSI flag if `prefers-reduced-motion` is enabled, since that will also "disable animations on the first page". I don't think it'd be accurate if some of the users that have the reduced motion activated count towards this flag being OFF.

Reply:
have you tested (1) or is it working thanks to the render service?

Reply:
Tested it, it works